cub/cub/device/device_memcpy.cuh

```diff
@@ -190,20 +190,22 @@ struct DeviceMemcpy
     // IDIV_CEIL(num_buffers, 64)
     using BlockOffsetT = uint32_t;
 
+    constexpr MemcpyOption memcpy_option = MemcpyOption::Memcpy;
+
     return detail::DispatchBatchMemcpy<
       InputBufferIt,
       OutputBufferIt,
       BufferSizeIteratorT,
       BufferOffsetT,
       BlockOffsetT,
       detail::batch_memcpy::policy_hub<BufferOffsetT, BlockOffsetT>,
-      true>::Dispatch(d_temp_storage,
-                      temp_storage_bytes,
-                      input_buffer_it,
-                      output_buffer_it,
-                      buffer_sizes,
-                      num_buffers,
-                      stream);
+      memcpy_option>::Dispatch(d_temp_storage,
```

Comment:
Suggestion: it would be more readable to me like this:
```c++
    return detail::DispatchBatchMemcpy<
      InputBufferIt,
      OutputBufferIt,
      BufferSizeIteratorT,
      BufferOffsetT,
      BlockOffsetT,
      detail::batch_memcpy::policy_hub<BufferOffsetT, BlockOffsetT>,
      MemcpyOption::Memcpy>::Dispatch(d_temp_storage,
```

Reply:
Agreed!